### ISSUE TITLE
Accessing a non public event URL with a "not logged in" user displays a white screen rather than a login UI

### ISSUE THREAD

**Issue by oseg** [iionly/event_calendar#5]
Accessing a non public event URL with a "not logged in" user displays a white screen rather than a login UI
This use case usually happens when clicking on a notification email hyperlink

Potential fix:

event_calendar v1.10.3, in start.php, add calls to:
- elgg_entity_gatekeeper and  elgg_group_gatekeeper at line 240

```
        case 'view':
+                        elgg_entity_gatekeeper($page[1], 'object', 'event_calendar');
+                        elgg_group_gatekeeper();
            echo event_calendar_get_page_content_view($page[1]);
```

**Comment by iionly** [comment]
Most likely the fix you have suggestion will work fine. As I'm currently on holiday I can test it out right now. I'll look into it once back home.

**Comment by iionly** [comment]
Fixed in version 2.3.1.
